CONTRIBUTING.md

```diff
@@ -1,19 +1,83 @@
-![repository-banner.png](https://res.cloudinary.com/alvarosaburido/image/upload/v1683452574/repo-banner_d2xeem.png)
+# Contributing Guide
```

Comment:
@alvarosabu I did a simple replace for now, but happy to adjust the document here and introduce repository banner and other paragraphs again which were previously there. :)

Reply:
I will do a separate commit with your suggested solution in the raised issue, so we have two options to consider :)

Reply:
I see, so if I understand you correctly you would prefer to keep the contribution markdown in the .github repository if we go for this approach and link their and remove the duplicate one in this repository. 
Also fine for me :) 
I will adjust the code before I do have a look at the other approach.

Reply:
addressed in de8a2a660a37e560c202e8af25258a6f95e2ec24